TESTING_GUIDELINES.md

```diff
@@ -12,10 +12,40 @@ Staging is redeployed everytime we merge to the `main` branch, so if you're look
 
 Once you have the application running, you can visit it in your browser at http://localhost:8443.
 
-You can also access it from other devices in your same network (like a mobile phone) for additional testing. See the [finding your local IP address](./README.md#finding-your-local-ip-address) section of the README for how to identify the local IP adress Nuxt is served on. Once you have identified your local IP address, you can access the website running on your computer by visiting `https://<local IP>:8443` replacing `<local IP>` (including the brackets) with the value you found using the instructions above in your mobile devices browser.
+You can also access it from other devices in your same network (like a mobile phone) for additional testing. See the [finding your local IP address](./README.md#finding-your-local-ip-address) section of the README for how to identify the local IP adress Nuxt is served on. Once you have identified your local IP address, you can access the website running on your computer by visiting `https://<local IP>:8443` replacing `<local IP>` (including the brackets) with the value you found using the instructions above in your mobile device's browser.
 
 Testing from multiple different devices as often as possible is a great way to contribute to Openverse's frontend development.
 
+### API Authentication
+
+By default, the application will run using the production API without authentication. This means your local frontend server may be subject to the unauthenticated rate limiting, depending on your testing behavior. There are two ways to solve this:
+
+1. You can change your local Nuxt server to point to a local API server that does not have throttling enabled
+2. You can introduce the environment variables necessary for authenticating with an API
+
+For the first, [run the Openverse API locally](https://github.com/WordPress/openverse-api). Then create a `.env` file with the following:
+
+```shell
+API_URL="http://localhost:8000/"
+```
+
+The run the dev server with environment loading:
+
+```shell
+pnpm dev:env
```

Comment:
Is this necessary? I was using and `.env` file before this PR, and variables were getting in fine.

Reply:
Oh really? I didn't realize but I guess Nuxt has it built in! Thanks for pointing it out. I will revise the instructions.

Reply:
Fixed in https://github.com/WordPress/openverse-frontend/pull/1569/commits/0976ebd6bfa10b7aa937c67307ca300755ed76cf

Thanks again!